File: Selenium/Project/LMS_Activity2.java
package project;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class LMS_Activity2 {

	public static void main(String[] args) {

		System.setProperty("webdriver.chrome.driver", "path/to/chromedriver");

		WebDriver driver = new ChromeDriver();

		try {

			driver.get("https://alchemy.hguy.co/lms");

			WebElement headingElement = driver.findElement(By.tagName("h1"));

			String headingText = headingElement.getText();

			if ("Learn from Industry Experts".equals(headingText)) {
				System.out.println("Heading matches successfully.");
			} else {
				System.out.println("Heading does not match. The actual heading is: " + headingText);
			}
		} catch (Exception e) {
			e.printStackTrace();
		} finally {

			driver.quit();
		}
	}
}

File: Selenium/Project/lms_Activity2.py
# Get the heading of the webpage.
# Goal: Make sure it matches “Learn from Industry Experts” exactly

# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

    # Start the Driver
with webdriver.Firefox() as driver:

    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/lms/")
    driver.maximize_window()

    # Get the heading of the webpage.
    page_heading = driver.find_element(By.XPATH, "//h1[contains(@class,'title')]")
    print ("The Page heading is : ",page_heading.text)

    # Verify the heading of the webpage.
    if(page_heading.text =="Learn from Industry Experts"):
        print ("The Page heading is : ",page_heading.text)
    else:
        print ("Wrong Website ") 

    #close the browser
    driver.quit()